packages/graph/hash_graph/lib/graph/src/store/mod.rs

```diff
@@ -332,9 +325,9 @@ pub trait Store {
     /// - if the [`LinkType`] doesn't exist.
     async fn update_link_type(
         &mut self,
-        property_type: LinkType,
+        property_type: &LinkType,
         updated_by: AccountId,
-    ) -> Result<Persisted<LinkType>, UpdateError>;
+    ) -> Result<(), UpdateError>;
 
     /// Creates a new [`Entity`].
     ///
```

Comment:
wanted to link to https://github.com/hashintel/hash/blob/fb29f412f60fda626e76300f0814dd6257a5348d/packages/graph/hash_graph/lib/graph/src/store/mod.rs#L387
and
https://github.com/hashintel/hash/blob/fb29f412f60fda626e76300f0814dd6257a5348d/packages/graph/hash_graph/lib/graph/src/store/mod.rs#L423

I think we should be consistent in the `Store` and also have these take in a reference and return unit. Thoughts on that?

Reply:
I adjusted the links as well, thanks for flagging!